.github/workflows/ci-release.yml

```diff
@@ -0,0 +1,73 @@
+name: CI Release
```

Comment:
change name to nightly-release.yml

Reply:
Name of workflow changed to nightly release (filename and `name` attribute inside the yml).